src/core/annotation.js

```diff
@@ -715,6 +715,8 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
     // Process field flags for the display layer.
     this.data.readOnly = this.hasFieldFlag(AnnotationFieldFlag.READONLY);
     this.data.multiLine = this.hasFieldFlag(AnnotationFieldFlag.MULTILINE);
+    this.data.comb = this.hasFieldFlag(AnnotationFieldFlag.COMB) &&
+                     this.data.maxLen !== null && !this.data.multiLine;
```

Comment:
Good point. I thought we didn't need it because we do not support the `Password` and `FileSelect` flags anyway, but it makes sense to add these checks anyway to prevent any issues. I'll update the patch.